### PR TITLE
docs(postman): include a section about authenticating postman with wo…

### PR DESCRIPTION
**Pull request by ethan92429**
…rking exported collection

### CODE REVIEW

File: postman/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Authenticating with Postman
```
Comment (mvonhasselbach) [inline postman/README.md:1]: Need another # for correct header formatting

File: postman/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+to accepted params, etc... Postman supports the Onshape OAuth2.0 authentication out of the box. 
+With a little tweaking, Postman can also support API key authentication.
+
+## API Key Authentication
+ 
+How to setup Postman to correctly sign API requests:
+1. File->Import-> select this file: [Onshape Postman Collection.](./onshape.postman_collection.json).
+1. Now make and keep handy your secret key and access key from the [Onshape Dev Portal](https://dev-portal.onshape.com/).
+1. Add the environment variables `accessKey` and `secretKey` to the collection as seen here:
+![open environment variables in collection](./postman_open_env_vars.jpg)
```
Comment (mvonhasselbach) [inline postman/README.md:13]: I'll provide a higher quality picture here (not sure if I can push it to the branch - I'll try).

Comment (ethan92429) [reply postman/README.md:13]: I think I gave you write access. Should work!

File: postman/README.md (hunk 1 above)
Comment (ethan92429) [reply postman/README.md:1]: I wanted an h1 header though. This is the title of the page.

**Pull request merged**